### PR TITLE
sql: improve DDL description

### PR DESCRIPTION
**Pull request by yikeke**
Via: https://github.com/pingcap/docs/pull/1097/ PTAL @lilin90 @winkyao

### CODE REVIEW

File: sql/mysql-compatibility.md [hunk 1]
```diff
@@ -77,45 +77,25 @@ TiDB 支持常用的 MySQL 内建函数，但是不是所有的函数都已经
...
 
-TiDB 实现了 F1 的异步 Schema 变更算法，DDL 执行过程中不会阻塞线上的 DML 操作。目前已经支持的 DDL 包括：
-
-+ Create Database
-+ Drop Database
-+ Create Table
-+ Drop Table
-+ Add Index
-+ Drop Index
+在 TiDB 中，运行的 DDL 操作不会阻止对表的读取或写入。但是，目前 DDL 变更有如下一些限制：
```
Comment (dcalvin) [inline sql/mysql-compatibility.md]:
```suggestion
在 TiDB 中，运行的 DDL 操作不会影响对表的读取或写入。但是，目前 DDL 变更有如下一些限制：
```

File: sql/mysql-compatibility.md [hunk 2]
```diff
@@ -77,45 +77,25 @@ TiDB 支持常用的 MySQL 内建函数，但是不是所有的函数都已经
...
-
-+ TiDB 的 `EXPLAIN` 命令返回的查询执行计划的输出与 MySQL 不同。更多内容参见[理解 TiDB 执行计划](../sql/understanding-the-query-execution-plan.md)。
+  - 不支持同时创建多个列 
+  - 不支持将新创建的列设为主键或唯一索引，也不支持将此列设成 auto_increment 属性
++ Drop Column: 不支持删除主键列或索引列
++ Change/Modify Column
+  - 不支持有损变更，比如从 `BIGINT` 变为 `INTEGER`，或者从 `VARCHAR(255)` 变为 `VARCHAR(10)`
+  - 不支持更改 `UNSIGNED` 属性
+  - 只支持将 `CHARACTER SET` 属性从 `utf8` 更改为 `utf8mb4`
++ `LOCK [=] {DEFAULT|NONE|SHARED|EXCLUSIVE}`: 支持的语法，但是不适用于 TiDB。所有支持的 DDL 变更都不会锁表。
```
Comment (dcalvin) [inline sql/mysql-compatibility.md]:
```suggestion
+ `LOCK [=] {DEFAULT|NONE|SHARED|EXCLUSIVE}`: TiDB 支持语法，但并不会生效。所有支持的 DDL 变更都不会锁表。
```

Comment (dcalvin) [reply sql/mysql-compatibility.md]: @yikeke PTAL

Comment (yikeke) [reply sql/mysql-compatibility.md]: Updated. PTAL again. @dcalvin

**Pull request closed without merging**